src/EnergyPlus/StandardRatings.cc

```diff
@@ -6469,7 +6469,7 @@ namespace StandardRatings {
                 } else if (bl >= q_full) {
                     // CASE 3 : 11.2.2.3.3 AHRI-2023
                     // Building Load is greater than the capacity of the unit at the Full Compressor Speed, q_full <= bl or (bl >= q_full:)
-                    if (t > (-15) || t <= (-8.33)) {
+                    if (t > (-15) || t <= (-8.33)) { // Logical disjunction always evaluates to true: t > -15 || t <= -8.33
```

Comment:
Didn't want to change the answer

[src/EnergyPlus/StandardRatings.cc:6472]:(warning),[incorrectLogicOperator],Logical disjunction always evaluates to true: t > -15 || t <= -8.33.